css/style.css

```diff
@@ -16,4 +15,110 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+body {
+  background-color: white;
+  margin: 0;
+  text-align: center;
+}
+
+header {
+  display: flex;
+  padding: 20px 0px;
+  justify-content: space-between;
+}
+
+header img {
+  height: 50px;
+  padding-left: 80px;
+}
+
+nav {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 100px;
+}
+
+ul {
+  display: flex;
+  list-style-type: none;
+  font-size: 18px;
+}
+
+li {
+  padding: 0 25px;
+}
+
+a {
+  color: #7f8c8d;
+  text-decoration: none;
+}
+
+a:hover {
+  color: #e74c3c;
+}
+
+.meet-karma {
+  font-weight: bold;
+}
+
+article {
+  padding: 100px;
+}
+
+.hero {
+  background: url(../img/first-background.jpg) no-repeat;
+  background-size: cover;
+  height: 250px;
+  color: white;
+  padding: 10% 0px;
+}
+
+button {
+  background-color: #e74c3c;
+  border-radius: 5px;
+  padding: 15px 25px;
+  margin-top: 30px;
+  border: none;
+}
+
+h1 {
+  font-weight: normal;
+  font-size: 50px;
+}
+
+p {
+  font-size: 20px;
+}
+
+.second-section {
+  padding-bottom: 0px;
+}
+
+.second-section-columns {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  border-bottom: 1px solid #bdc3c7;
+  margin-top: 50px;
+  padding-bottom: 100px;
+}
+
+.second-section-columns img {
+  width: 100px;
+}
 
+footer p {
+  font-size: 15px;
+  padding-bottom: 10px;
+}
+
+footer img {
+  width: 15px;
+  border: 1px solid #bdc3c7;
+  border-radius: 100px;
```

Comment:
Your icons are being rounded by the border. Don't apply the border directly on the image - instead, wrap it inside a `div` and apply the border and border-radius to that.

Reply:
Thanks, border and border-radius added to div rather than on image as suggested